dev/io.openliberty.mcp/src/io/openliberty/mcp/tools/ToolResponse.java

```diff
@@ -65,6 +65,18 @@ public static ToolResponse error(String message) {
         return new ToolResponse(true, List.of(new TextContent(message)));
     }
 
+    /**
+     * Creates an unsuccessful response using the provided Throwable.
+     * If the exception has a message, it will be used; otherwise, the class name of the Throwable is used.
+     *
+     * @param t the Throwable representing the error
+     * @return a ToolResponse marked as error, containing the extracted message
+     */
+    public static ToolResponse error(Throwable t) {
+        String message = (t.getMessage() != null) ? t.getMessage() : t.getClass().getSimpleName();
+        return error(message);
+    }
+
```

Comment:
The code here is fine, but since we want to stay aligned with the quarkus-mcp-server API, please can we not make changes to this class.